packages/GA4Client/src/common.js

```diff
@@ -132,13 +132,15 @@ Common.prototype.standardizeParameters = function (parameters) {
 };
 
 Common.prototype.standardizeName = function (name) {
-    try {
-        name = window.GoogleAnalytics4Kit.setCustomNameStandardization(name);
-    } catch (e) {
-        console.error(
-            'Error calling setCustomNameStandardization callback. Check your callback.  Data will still be sent without user-defined standardization. See our docs for proper use - https://docs.mparticle.com/integrations/google-analytics-4/event/',
-            e
-        );
+    if (window.GoogleAnalytics4Kit.setCustomNameStandardization(name)) {
```

Comment:
If you're checking if `setCustomNameStandardization` exists, you might be better off using `hasOwnProperty` instead of calling the function directly.

If the function does not exist, calling the function would throw an error before the try/catch block. Also, if the function does exist, you're essentially calling it twice.

```suggestion
    if (window.GoogleAnalytics4Kit.hasOwnProperty(setCustomNameStandardization)) {
```

Reply:
@alexs-mparticle thanks for correction, applying the change